helpers/club.js

```diff
@@ -1,13 +1,61 @@
-const studentModel = require('../models/student')
+const clubModel = require('../models/club')
+const jwt=require("jsonwebtoken");
+const imageFunction = require("../helpers/image");
+
 
 exports.editProfile = async function (req, res, next) {
-    res.status(501).json({
-        message: "Not implemented yet"
-    })
+    const {name,tags,description,profilePicURL,coverPicURL,website}=req.body;
+    editableFields={name,tags,description,profilePicURL,coverPicURL,website};
+    const token=req.headers.authorization.split(" ")[1];
+    const decodedToken=await jwt.verify(token,req.app.get('secretKey'));
```

Comment:
Is there a reason you're not using the auth middleware, and doing the verification inside the functions themselves?